internal/util/get/get.go

```diff
@@ -98,9 +99,9 @@ func (c Command) Run(ctx context.Context) error {
 		return cleanUpDirAndError(c.Destination, err)
 	}
 
-	if err := addmergecomment.Process(c.Destination); err != nil {
-		return cleanUpDirAndError(c.Destination, err)
-	}
+	_ = addmergecomment.Process(c.Destination)
+	t := usage.Tracker{PackagePaths: []string{"c.Destination"}}
+	t.TrackAction("pkg")
```

Comment:
I am assuming TrackAction is reading and writing the resources. Can we combine both (add-merge-key, track-action) in one pipeline operation to avoid reading/writing resources again ?

It's like post filters after a pkg fetch.

Reply:
Agree. It would reduce the IO overhead. Updated.